bitnami/kiam/templates/agent/agent-service.yaml

```diff
@@ -43,6 +49,9 @@ spec:
       {{- else if eq .Values.agent.service.type "ClusterIP" }}
       nodePort: null
       {{- end }}
+    {{- if .Values.service.agent.extraPorts }}
```

Comment:
These two values are reversed, it should be `.Values.agent.service.extraPorts`

Reply:
Thank you @voor, I have fixed this in this PR: https://github.com/bitnami/charts/pull/10130